components/automate-es-gateway/habitat/default.toml

```diff
@@ -48,3 +48,5 @@ tcp_nopush = "on"
 tcp_nodelay = "on"
 ssl_verify_depth = 2
 proxy_connect_timeout = 5
+proxy_read_timeout = 600
+proxy_send_timeout = 600
```

Comment:
I know this is annoying, but I think for hab config items that are also user-configurable, we should probably set the default in the config_request.go file as well for consistency with the rest of the configuration?